src/sharedAPI.ts/apiQueries.ts

```diff
@@ -18,7 +18,7 @@ export const getComicBySlug = async (slug: string): Promise<MangaDetails> => {
   return await comic.json();
 };
 
-export const getComicChapters = async (hid: string): Promise<Chapters> => {
+export const getComicChapters = async (hid: string | undefined): Promise<Chapters> => {
```

Comment:
Just preference but you can also just do `hid?: string`

Reply:
Noted